src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -71,7 +72,12 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
   };
 
   return (
-    <div className={`${styles.expandedRow}`}>
+    <Box
+      id='revision-row-expandable'
```

Comment:
I have a few comments :)

1. the most important: the id needs to be unique, here it's not unique! You need to understand that it's currently the same for each line. In this case, because we don't have a unique id or a key for a result, I believe the easiest way would be the `useId` hook (see https://react.dev/reference/react/useId). You can call it in `RevisionRow` and pass it to `RevisionRowExpandable`. You'll also need to do the same in `SubtestsRevisionRow` BTW.
2. you don't need a `Box`, just use a `section` element directly. (I understand you picked the `section` so that you can add an `aria-label`. I'm not fully sure it's useful but why not...)